src/scss/components/Card-mobile.scss

```diff
@@ -0,0 +1,99 @@
+@use "../abstracts/fonts.scss";
+@use "../abstracts/colors.scss";
+@import "../abstracts/breakpoints.scss";
+
+.product-card {
+  align-items: flex-start;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  height: 22.1rem;
+  margin: 5rem 0;
+  padding: 10px;
+  width: 20rem;
+  transition: 0.1s ease-in;
+
+  &:hover {
+    cursor: pointer;
+    background-color: colors.$productCardColor;
+  }
+
+  &--active {
+    outline: 1px solid colors.$productCardColorBlueBorder;
+  }
+
+  &--active-error {
+    outline: 1px solid colors.$productCardColorRedBorder;
+  }
+
+  &--disabled {
+    opacity: 0.4;
+  }
+
+  &--disabled:hover {
+    cursor: initial;
+    background-color: transparent;
+  }
+
+  .product-card__picture {
+    border: 0;
+    height: 100px;
+    justify-self: center;
+    margin-bottom: 2.1rem;
+    padding: 0;
+    width: 100%;
+  }
```

Comment:
Brakuje `border-radius` - nie powinieneś polegać na tym, że zawsze zostanie przekazany obrazek z zaokrąglonymi rogami. Szczególnie, że to wymaga przekazywania PNG-ów z przeźroczystością, którą są "cięższe" od JPG-ów, które by tutaj zupełnie wystarczyły :)